src/main/java/org/killbill/billing/overdue/api/OverdueCondition.java

```diff
@@ -35,4 +35,6 @@
     public PaymentResponse [] getResponseForLastFailedPaymentIn();
 
     public ControlTagType getControlTagType();
```

Comment:
done!

> Let's not forget to update the release notes ;-)

Is [this section](https://github.com/killbill/killbill/wiki/Kill-Bill-Migration-0.16.x-to-0.18.x#java-api-changes) where I have to add this issue changes?